sdk/core/Azure.Core.TestFramework/src/TestEnvironment.cs

```diff
@@ -131,7 +131,7 @@ static TestEnvironment()
         /// <summary>
         ///   The URL of the Azure Authority host to be used for authentication. Recorded.
         /// </summary>
-        public string AuthorityHostUrl => GetRecordedOptionalVariable("AZURE_AUTHORITY_HOST");
+        public string AuthorityHostUrl => GetRecordedOptionalVariable("AZURE_AUTHORITY_HOST") ?? "https://login.microsoftonline.com/";
```

Comment:
Why do we need this fallback? Net-TestResources should provide a value.

Reply:
When running in public, TestEnvironment.AuthorityHostUrl is null, AZURE_AUTHORITY_HOST doesn't set in SubscriptionConfiguration: $(sub-config-azure-cloud-test-resources)

Reply:
Is there still an issue sourcing AZURE_AUTHORITY_HOST from the public subscription configuration?

Reply:
Yes, if removing the hard code `?? AzureAuthorityHosts.AzurePublicCloud.ToString()` , it will cause some jobs fail.